MAVProxy/mavproxy.py

```diff
@@ -538,7 +538,10 @@ def process_stdin(line):
 
     if cmd == 'help':
         k = command_map.keys()
-        k.sort()
+        if sys.version_info.major < 3:
+            k.sort()
+        else:
+            sorted(k)
```

Comment:
I thought sorted() returns a new list? Can be use:
 k = sorted(k)
for both versions?

Reply:
Fixed - using ``k = sorted(k)``